CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+# Contributor Guide
+
+This library is open source and under the ([GNU General Public License v3.0][LICENSE]), 
+which will cover any contributions you may make to this project. 
+If you're interested in contributing to the ``juju-verify`` code or documentation, 
+the following will help you get started. Please familiarise yourself with the terms of
+the license and read this page before start working on any contributions.
+
+## Python version
+
+This tool has been supported since Pythone version 3.6.
+
+## Code of conduct
+
+We have adopted the [Ubuntu Code of Conduct][COC].
+
+## Releases and versions
+
+The juju-verify uses semantic versioning three-part version number, where the numbers
+describe:
+
+1. major - version of ``juju`` with which it is compatible
```

Comment:
I'm not sure I understand this. If juju-verify works for minimum 2.8.10, does this mean that juju-version version will be 2.x.x? What if the min version would be 2.9? It would be clear is the min version would be Juju 3.0, whenever it is released.

I think we hould follow around own versioning, starting in 1.0 for stable releases, and maybe releasing 2.0 when kubernetes verifiers are included (right now, only openstack-related verifiers exist).

Reply:
Yes, you are right. In my head I had the idea that the current version is 2.2.1, so I assumed that the first number represents the first number in the juju version.